apps/site/src/components/pages/blocks/block-form-styles.tsx

```diff
@@ -1,17 +1,17 @@
-import { experimental_sx as sx, Paper, styled } from "@mui/material";
+import { Paper, styled } from "@mui/material";
 
-export const BlockFormContainer = styled(Paper)(
-  sx({ flex: 1, width: "100%", p: 6 }),
+export const BlockFormContainer = styled(Paper)(({ theme }) =>
+  theme.unstable_sx({ flex: 1, width: "100%", p: 6 }),
 );
```

Comment:
This is a consequence of updating to MUI 5.11.x, which we also did in `@hashintel/type-editor`